lib/posts.js

```diff
@@ -220,4 +221,48 @@ PostsRequest.prototype.statuses = function() {
 	return this;
 };
 
+/**
+ * @method year
+ * @chainable
+ * @param year - Integer for year
+ * @returns {CollectionRequest} The PostsRequest instance (for chaining)
+ */
+PostsRequest.prototype.year = function ( year ) {
+  return this.filter( 'year', year );
+};
+
+/**
+ * @method month
+ * @chainable
+ * @param month - Integer for month or month string ("january")
```

Comment:
Re: the prevous comment, you can specify multiple types as `@param {Number|String} month Integer for month or month string e.g. "January"`